src/display/text_layer.js

```diff
@@ -63,6 +63,7 @@ var renderTextLayer = (function renderTextLayerClosure() {
     // Initialize all used properties to keep the caches monomorphic.
     var textDiv = document.createElement('div');
     var textDivProperties = {
+      style: '',
```

Comment:
I suppose that we can use `style: null,` instead, since the only case where it's not going to be set should be the `isWhitespace` case that we're now correctly skipping anyway.

However, if you use `originalTransform: null,` as well, then you'd need to do as in the snippet below to prevent any issues. Is that what you meant here?
```javascript
CustomStyle.setProp('transform', div,
                    divProperties.originalTransform || '');
```